ui/src/app/views/cohort-builder/search/cohort-builder/cohort-builder.component.css

```diff
@@ -4,10 +4,10 @@
 }
 
 .my-border {
```

Comment:
i guess we should call this something other than my-border. We can take care of that.